### PR TITLE
Fixing stuff

### LINKED ISSUE #6219: Phasic Scanning Modules
(WEB REPORT BY: iagfprmf REMOTE: 158.69.55.2:7777)
Phasic scanning modules cannot be used in place of advanced modules in mechs, even though phasic is a better part.

### LINKED ISSUE #6243: Catwalks are not seen as valid flooring for a variety of things.
@dylanstrategie.

### LINKED ISSUE #6271: IAA False-Bottom Suitcase eats items
Once you have the false bottom open, you can keep sticking items into the false bottom, but only the last item inserted can be retrieved. Everything else is gone forever.

### PR DESCRIPTION
**Pull request by DeityLink**
Fixes #6243
Fixes #6271
- You can no longer place something in a briefcase's fake bottom if there's already something in it.
- Removed the "on_floor" requirement to build computer frames, stools, closets, canisters and iv_drips.
- Empty donut boxes, egg boxes, and the likes, can now be unfolded back into cardboard sheets.

I also made sheet_types.dm marginally easier to read and edit.

### CODE REVIEW

**Comment by clusterfack** [comment]
That's not a fix to that bug, don't just remove the component requirement because that's literally fucking retarded

**Comment by DeityLink** [comment]
Aight

**Comment by clusterfack** [comment]
Also removing the on floor requirement lets you build shit in space which is also shit

**Comment by DeityLink** [comment]
@clusterfack I only removed it for a few specific objects that aren't anchored by default.

**Comment by Kurfursten** [comment]
This is a buff to engineers, now you don't have to walk back to the airlock to build canisters

File: code/game/objects/items/weapons/storage/briefcase.dm [hunk 1]
```diff
@@ -80,7 +80,12 @@
 
 	var/busy_hunting = 0
 	var/bottom_open = 0 //is the false bottom open?
-	var/obj/item/stored_item //what's in the false bottom. If it's a gun, we can fire it
+	var/obj/item/stored_item = null //what's in the false bottom. If it's a gun, we can fire it
+
+/obj/item/weapon/storage/briefcase/false_bottomed/Destroy()
+	if(stored_item)//since the stored_item isn't in the briefcase' contents we gotta remind the game to delete it here.
+		qdel(stored_item)
```
Comment (clusterfack) [inline code/game/objects/items/weapons/storage/briefcase.dm]: stored_item = null after

**Comment by DeityLink** [comment]
should be good now.

**Pull request merged**